board/app/models/follow.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class Follow extends AppModel 
+{
+
+    public function addFollowing()
+    { 
+        try {
+            $db = DB::conn();
+            $db->begin();
+            $params = array(
+            'username' => $this->username,
+            'user_id' => $this->user_id
+            );
```

Comment:
Please apply fixing of indentions for array declaration, with the same comment as this one:
https://github.com/maragatchalian/FoodFacts/pull/2/files#r30455780

(Apply to all others :dart: )